gass/copy/source/globus_gass_copy_glob.c

```diff
@@ -2175,7 +2202,7 @@ globus_l_gass_copy_cksm_file(
     *cksmptr = '\0';
     
     strncpy(cksm, cksm_buff, sizeof(cksm_buff));
-    
+    //globus_libc_fprintf(stderr,"SPB file-checksum %s=%s\n",url,cksm); 
```

Comment:
I think we probably shouldn't put (commented-out) debug statements in the code. Although it of course can be very useful.